scripts/releases/ios-prebuild/configuration.js

```diff
@@ -48,13 +48,14 @@ const dependencies /*: $ReadOnlyArray<Dependency> */ = [
         'src/vlog_is_on.cc',
       ],
       headers: ['src/glog/*.h'],
-      // resources: ['../third-party-podspecs/glog/PrivacyInfo.xcprivacy'],
+      resources: ['../third-party-podspecs/glog/PrivacyInfo.xcprivacy'],
       headerSkipFolderNames: 'src',
     },
     settings: {
       publicHeaderFiles: './headers',
       headerSearchPaths: ['src'],
-      compilerFlags: ['-Wno-shorten-64-to-32', '-Wno-everything'],
+      cCompilerFlags: ['-Wno-shorten-64-to-32', '-Wno-everything'],
+      cxxCompilerFlags: ['-Wno-shorten-64-to-32', '-Wno-everything'],
```

Comment:
Can we remove the `-Wno-everything`?

Reply:
this silences all the warning and we might want to have some visibility over them.

Reply:
We also miss the C++ language standard. I think we should apply it to all the pods.

Reply:
Agree x 2 - will fix.